src/rpc/rawtransaction.cpp

```diff
@@ -294,6 +294,8 @@ UniValue verifytxoutproof(const JSONRPCRequest& request)
             "    \"blockhash\"   (string) The blockhash the included proof commits to\n"
             "    [\"txid\"]      (array, strings) The txid(s) which the proof commits to\n"
             "}\n"
+            "\nResult: (DEPRECATED. To see this result in v0.17 instead, please start bitcoind with -deprecatedrpc=verifytxoutproof. Empty if proof is invalid)\n"
```

Comment:
I'm not sure about the `in v0.17 instead` wording -- what about when bitcoin hits v0.18?

Reply:
I thought the same, but there are other occurrences in the code. I would remove the version though.

Reply:
I'm a bit confused, if this is merged now, it definitely would be the way to see the old text for 0.17. Now, perhaps the note should also say when it will be removed?

Reply:
Pushed a new wording, hopefully clearer and more pertinent to the user.